lib/siwapp_web/live/customer_live/index.html.heex

```diff
@@ -0,0 +1,28 @@
+<h1> <%= @page_title %> </h1>
+<table class="table is-bordered is-striped is-hoverable is-fullwidth">
+    <thead>
+      <tr>
+        <th class="no-wrap" title="Customer name"> Customer name </th>
+        <th class="no-wrap" title="Identification"> Identification </th>
+        <th class="text-right" title="Due"> Due </th>
+        <th class="text-right" title="Total"> Total </th>
+        <th class="text-right" title=" "></th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <%= for customer <- @customers do%>
+        <%=if customer.active do %>
```

Comment:
I think we'll probably remove the active field in customers... so you can remove this